tests/unit/mock/config/parser/base/adva_fsp150f2/adva_fsp150f2_full_received.py

```diff
@@ -24,28 +24,35 @@
         config_line='  security-banner "Any changes to this system require a change ticket or trouble ticket.  Do you really want to proceed?\\\\n\\\\nBe sure to notify the XNOC of work being performed BEFORE changes are made.\\\\n\\\\nXNOC - 888-555-1963"',
         parents=("configure system",),
     ),
-    ConfigLine(config_line="  pae authentication-control enable", parents=()),
+    ConfigLine(config_line="  pae authentication-control enable", parents=("configure system",)),
+    ConfigLine(config_line="exit", parents=()),
```

Comment:
I don't think there should ever be a case where a `ConfigLine()` is exactly the same. I could be wrong about this, but would want to make sure we are all in sync.